src/main/scala/io/buoyant/linkerd/RateLimiterConfig.scala

```diff
@@ -0,0 +1,23 @@
+package io.buoyant.linkerd
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.finagle.{Filter, Stack}
+import io.buoyant.linkerd.protocol.HttpRequestAuthorizerConfig
+
+class RateLimiterConfig(limit: Int, window: Int) extends HttpRequestAuthorizerConfig{
```

Comment:
Should `window` include a unit in its name (like other time values in linkerd)?

Reply:
Should we make the value `window` optional so that it defaults to 1 second?

Reply:
I changed `window` to `intervalSeconds` to reflect the naming convention of time values in linkerd. It has a default value of 1 now. `limit` still must be passed.

Reply:
The bounds checks and default value occur in `RateLimiterConfig` since that is the first place we can check these values when a user creates a new `RateLimiter`. Correct me if this is an incorrect assumption, but we don't expect users to interact with plugins by creating a `new RateLimiter(...)` themselves?